src/views/Search.vue

```diff
@@ -4,15 +4,11 @@
       <v-row align="center" justify="center" class="d-inline d-sm-flex">
         <v-card style="text-align: center" elevation="1" class="mb-3 mx-2 px-3">
           <b>search for artist</b><br/>
-          <v-text-field dense solo flat hide-details class="centered-input" v-model="query" label="research query" append-outer-icon="mdi-close" @click:append-outer="query = ''" prepend-icon="mdi-magnify" @click:prepend="search" />
+          <v-text-field dense solo flat hide-details class="centered-input" v-model="query" label="research query" append-outer-icon="mdi-close" @click:append-outer="query = ''" prepend-icon="mdi-magnify" @click:prepend="search" @keyup.enter="search" />
```

Comment:
would be better to have the state in the url ?query=, and have it as the single source of truth, would make sharing more consistent and easy